01_part-to-whole_cnell/_targets.R

```diff
@@ -0,0 +1,49 @@
+library(targets)
+library(tidyverse)
+library(rvest)
+library(spData)
+library(sf)
+library(cartogram)
+library(showtext)
+library(scico)
+library(gganimate)
+
+options(tidyverse.quiet = TRUE)
+
+source("src/data_utils.R")
+source("src/plot_utils.R")
+
+wss_url <- 'https://www.usgs.gov/special-topics/water-science-school/science/how-wet-your-state-water-area-each-state'
+proj <- "+proj=lcc +lat_1=30.7 +lat_2=29.3 +lat_0=28.5 +lon_0=-91.33333333333333 +x_0=999999.9999898402 +y_0=0 +ellps=GRS80 +datum=NAD83 +to_meter=0.3048006096012192 +no_defs"
```

Comment:
For a cartogram it would be best to use a equal area projection like [this](https://epsg.io/102008) rather than a conformal projection -- though I'd adjust that North American projection for CONUS, something like this (not tested):

`+proj=aea +lat_1=29.5 +lat_2=45.5 +lat_0=37.5 +lon_0=-96 +x_0=0 +y_0=0 +ellps=GRS80 +datum=NAD83 +units=m no_defs`

Reply:
that solved the issue with NY also!
![water_area](https://user-images.githubusercontent.com/17803537/160903653-b8f5008a-66df-4a81-9b44-258cb884b249.gif)

Reply:
Yay - awesome!! Just noticed that three other little odd things still seem to be happening: 
  * part of western MA breaks off and moves east during the transition
  * Virginia is breaking in half, and the eastern chunk disappears
  * Maryland is breaking in half, and the eastern chunk disappears

Reply:
hmm yeah I am not exactly sure what is doing that. If I increase the number of iterations given as an input to `cartogram_cont` I get slightly different versions, but still with some states getting split. I expect this might be part of the underlying function to find the optimal interpolation between sf objects